src/app/middleware/middlewares/ip-filter.middleware.ts

```diff
@@ -16,7 +16,7 @@ export class IpFilterMiddleware implements NestMiddleware {
 
   async use(request: any, res: any, next: (error?: any) => void): Promise<void> {
     const ip = request.headers['x-forwarded-for'] ?? request.connection.remoteAddress;
-    const userUuid = request.user.uuid;
+    const userUuid = request.user?.uuid;
```

Comment:
I think that await this.userRepository.update must be executed just if userUuid is present